pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.isort]
 profile = "black"
-src_paths = ["pytest_tests", "robot"]
+src_paths = ["dynamic_env_pytest_tests", "pytest_tests", "robot"]
```

Comment:
By the way, maybe delete "robot"?

Reply:
current tests with old dev-env will be broken then